### PR TITLE
Don't crash when attempting scratch/simple build

### PR DESCRIPTION
**Pull request by mmilata**
Related to #429

### CODE REVIEW

File: osbs/build/build_request.py [hunk 1]
```diff
@@ -794,12 +802,6 @@ def render(self, validate=True):
             self.spec.validate()
         super(ProductionBuild, self).render()
 
-        repo_name = git_repo_humanish_part_from_uri(self.spec.git_uri.value)
-        # NOTE: Since only the repo name is used, a forked repos will have
-        # the same git-repo-name tag. This is a known limitation. If this
-        # use case must be handled properly, the git URI must be taken into
-        # account.
-        self.set_label('git-repo-name', repo_name)
         self.set_label('git-branch', self.spec.git_branch.value)
```
Comment (lcarva) [inline osbs/build/build_request.py]: Why not move this one?

Comment (mmilata) [inline osbs/build/build_request.py]: Branch is only mandatory for production builds, though I can't remember why it's the case, maybe we can make it mandatory for all builds?

**Comment by twaugh** [comment]
We've talked about getting rid of build types before. I wonder if we should change Simple to Scratch?

**Comment by mmilata** [comment]
@twaugh I'm working on that but it's not ready yet - I realized we probably need to distinguish build types at least for the purpose of input validation (e.g. prod build needs the sources command but simple build doesn't). Or should this be the responsibility of koji-containerbuild?

This patch is supposed to un-break simple builds in the meantime.

**Pull request closed without merging**